server/athenian/api/openapi/openapi.yaml

```diff
@@ -1056,3 +1087,167 @@ components:
+        account:
+          description: Session account ID.
+          type: integer
+        date_from:
+          description: PRs must be updated later than or equal to this date.
+          format: date
+          type: string
+        date_to:
+          description: PRs must be updated earlier than or equal to this date.
+          format: date
+          type: string
+        in:
+          $ref: '#/components/schemas/RepositorySet'
+        stages:
+          items:
+            $ref: '#/components/schemas/PullRequestPipelineStage'
+          type: array
+        with:
+          $ref: '#/components/schemas/PullRequestWith'
+      required:
+      - account
+      - date_from
+      - date_to
+      - stages
+      type: object
+    PullRequestPipelineStage:
+      description: One of the four possible PR states of the modelled lifecycle.
+      enum:
+      - wip
+      - review
+      - merge
+      - release
+      # PR afterlife - PRs that were released and nothing ever-ever happens to them
+      - gold
+      example: wip
+      type: string
+    PullRequestSet:
+      description: List of pull requests.
+      items:
+        $ref: '#/components/schemas/PullRequest'
+      type: array
+    PullRequest:
+      description: Details of a pull request.
+      example:
+        # https://github.com/src-d/go-git/pull/1152
+        repository: github.com/src-d/go-git
+        number: 1152
+        title: 'Fix an edge case when fetching remotes with unrelated incomplete history.
+          Fixes #1151'
+        size_added: 37
+        size_removed: 1
+        files_changed: 2
+        created: 2019-05-17T01:14:00Z
+        updated: 2019-09-18T10:49:00Z
+        stage: wip
+        participants:
+        - id: github.com/NonLogicalDev
+          status: author
+        - id: github.com/mcuadros
+          status: commenter
+      properties:
+        repository:
+          description: PR is/was open in this repository.
+          type: string
+        number:
+          description: PR number.
+          type: integer
+        title:
+          description: Title of the PR.
+          type: string
+        size_added:
+          description: Overall number of lines added.
+          type: integer
+        size_removed:
+          description: Overall number of lines removed.
+          type: integer
+        files_changed:
+          description: Number of files changed in this PR.
+          type: integer
+        created:
+          description: When this PR was created.
+          format: date-time
+          type: string
+        updated:
+          description: When this PR was last updated.
+          format: date-time
+          type: string
+        stage:
+          $ref: '#/components/schemas/PullRequestPipelineStage'
+        participants:
+          description: List of developers related to this PR, always including the
+            author.
+          items:
+            $ref: '#/components/schemas/PullRequestParticipant'
```

Comment:
Why not using the same structure as `PullRequestWith`? That way participants are also grouped by role.

Reply:
I will add the profile images there.